kubernetes/platform-services/values.yaml

```diff
@@ -1,3 +1,26 @@
+merge-query-services: 
```

Comment:
Have the comments that these vars work in either or. So, we can't set both as `true`

Reply:
Let me create a pr for it